plugins/wazo-gigaset/common/common.py

```diff
@@ -131,6 +131,14 @@ def path_preprocess(self, request):
 class BaseGigasetPlugin(StandardPlugin):
     _ENCODING = 'UTF-8'
 
+    _LOCALE = {
+        'de_DE': '0',
+        'en_US': '1',
+        'es_ES': '6',
+        'fr_FR': '7',
+        'fr_CA': '0',
```

Comment:
fr_CA should be the same as en_US (`1`) - the phone system is practically the same.
Also, I'm not sure that LOCALE is the right variable name for this information, given that it is used for the country tone.

Reply:
Done.